source/helpers/MessageHelper.ts

```diff
@@ -14,6 +25,7 @@ const messages = {
   unknownError: "Okänt fel. Försök igen.",
   userNotFound:
     "Du har tyvärr inte åtkomst till tjänsten. För vidare hjälp ring kontaktcenter på 042-10 50 00.",
+  unkownError: "Okänt fel. Försök igen.",
```

Comment:
Remove this typo (and replace instances of `"unkownError"` with `"unknownError"`).